sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -939,6 +939,18 @@ def _add_argparse_args(cls, parser):
         help=
         'Create metrics reporting the approximate number of bytes written per '
         'bucket.')
+    parser.add_argument(
+        '--no_gcsio_throttling_counter',
+        default=False,
+        action='store_true',
+        help='Throttling counter in GcsIO is enabled by default. Set '
+        '--no_gcsio_throttling_counter to avoid it.')
+    parser.add_argument(
+        '--enable_gcsio_blob_generation',
```

Comment:
the difference of `bucket.blob` and `bucket.get_blob` is that the latter will send an API request which has cost. I remember we had concern about excessive API request before - #30205 

Possibly add a disclaimer that this will be at the cost of sending API request verify blob existence?

Reply:
> the difference of bucket.blob and bucket.get_blob is that the latter will send an API request which has cost. I remember we had concern about excessive API request before - https://github.com/apache/beam/pull/30205
Thats' right. I had that in the commit message https://github.com/apache/beam/pull/32428/commits/f82230d3b52f3bc2b8bbbd92d5fe33d3aa1daf0c,

> Possibly add a disclaimer that this will be at the cost of sending API request verify blob existence?
Good call. I think I can make that clear in the help message of this option.

Reply:
Fixed.